cabal.config

```diff
@@ -0,0 +1,2395 @@
+-- NOTE: Due to revisions, this file may not work. See:
```

Comment:
This file is really strange... How did you create it?

Reply:
It was created automatically during `stack new`.
It doesn't seem to affect anything, I'll remove it.